core/blockchain_test.go

```diff
@@ -3056,10 +3055,7 @@ func TestDeleteRecreateSlots(t *testing.T) {
 	// Import the canonical chain
 	diskdb := rawdb.NewMemoryDatabase()
 	gspec.MustCommit(diskdb)
-	chain, err := NewBlockChain(diskdb, nil, params.TestChainConfig, engine, vm.Config{
-		Debug:  true,
-		Tracer: logger.NewJSONLogger(nil, os.Stdout),
```

Comment:
I think it's not important for the test, but I suspect it was used during development of the test. It can be pretty tricky to construct a non-trivial blockchain test with contract interaction without ability to trace. 
So I don't think the change here specifically matters, however, this circular dependency might bite us later... ?

Reply:
I managed to avoid the circular dependency and reverted these changes. Package `logger` doesn't import `core` anymore